core/src/test/java/org/apache/accumulo/core/file/rfile/RFileTest.java

```diff
@@ -1512,6 +1512,39 @@ public void testReseekUnconsumed() throws Exception {
     trf.closeReader();
   }
 
+  @Test
+  public void testEstimateOverlappingEntries() throws IOException {
+    TestRFile trf = new TestRFile(conf);
+    // lower block sizes so estimates are closer
+    trf.openWriter(true, 100, 100);
+
+    // generate 1024 entries
+    int count = 0;
+    for (int row = 0; row < 4; row++) {
```

Comment:
Could this code to write data be pushed up into AbstractRfileTest so it could be shared w/ FencedRfileTest?

Reply:
We could, the only reason why I didn't was that the fencing test already has something to do a similar thing but I could likely refactor it to have it shared in the Abstract test (or at least try to)

Reply:
Please ignore the comment if it does not make sense.  I was not sure about the class hierarchy w/ the test and code sharing.

Reply:
I think i'll leave things how they are for now since the fenced test uses another method already to write the test data